package.json

```diff
@@ -7,6 +7,7 @@
   },
   "dependencies": {
     "@rails/webpacker": "^5.2.1",
+    "accessible-autocomplete": "^2.0.3",
```

Comment:
What widget is this js for?

Reply:
This https://miro.com/app/board/o9J_ldVNkCY=/?moveToWidget=3074457353712710918&cot=14